Libplanet/Net/Swarm.cs

```diff
@@ -597,9 +593,9 @@ public void BroadcastTxs(IEnumerable<Transaction<T>> txs)
 
             try
             {
-                await SyncBehindsBlocksFromPeersAsync(
+                await SyncBehindsBlocksFromPeerAsync(
                     workspace,
-                    peersWithHeight,
+                    peersWithHeight.First(),
```

Comment:
Is `peersWithHeight` sorted? (or, is `.First()` just arbitrary?)

Reply:
`peersWithHeight` is sorted with `Distance`, which is  determined by Kademlia protocol. So `.First()` will return closest peer from the caller.